reporting-guidelines.html

```diff
@@ -50,7 +50,7 @@
         <li><a href="/index.html#projects">Projects</a></li>
         <li><a href="/index.html#internship-opportunities">Internship Opportunities</a></li>
         <li><a href="/index.html#welcome-newcomers">Welcome Newcomers</a></li>
-        <li class="toggle"><a href="#main-menu">About Us <span class="fas fa-chevron-down" "arrow"></i></a>
+        <li class="toggle"><a href="#main-menu">About Us<i> <span class="fas fa-chevron-down" "arrow"></i></a>
```

Comment:
@prssanna minor nitpicking, but to be consistent with other `<i><span>` tags, we can remove the space between `<i>` and `<span>`

Reply:
@sara-02 Sorry about that, I've updated it! 👍